### PR TITLE
Wip rgw openssl

### PR DESCRIPTION
**Pull request by yehudasa**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -41,6 +41,7 @@ Build-Depends: autoconf,
                libleveldb-dev,
                libnss3-dev,
                libsnappy-dev,
+	       libssl-dev,
```
Comment (mattbenjamin) [inline debian/control]: spacing

**Comment by mattbenjamin** [comment]
is this safe to merge?

**Comment by yehudasa** [comment]
@mattbenjamin failed build:

http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-tarball-trusty-amd64-cmake/log.cgi?log=aaa0f6fa7798879160fb8685aef88d53ea8c7435

**Comment by alimaredia** [comment]
@mdw-at-linuxbox I just tried to build master with automake and it wouldn't compile due to a missing header (openssl/ssl.h).  I think this is because configure didn't seem to check for openssl-devel. Has anyone else particiapting in the PR notice that? If not remind me to talk to you about it later today.

**Pull request merged**